monitoring-custom-metrics-kyma1/README.md

```diff
@@ -0,0 +1,63 @@
+# Expose Custom Metrics in Kyma
+
+## Overview
+
+This example shows how to expose custom metrics to Prometheus with a Golang service in Kyma. To do so, follow these steps:
+
+1. Expose a sample application serving metrics on the `8081` port.
+2. Access the exposed metrics in Prometheus.
+
+## Prerequisites
+
+- Kyma as the target deployment environment.
+
+## Installation
+
+### Expose a sample metrics application
+
+1. Export your Namespace as a variable. Replace the `{namespace}` placeholder in the following command and run it:
+
+    ```bash
+    export KYMA_EXAMPLE_NS="{namespace}"
+    ```
+
+2. Deploy the service:
+
+    ```bash
+    kubectl apply -f deployment/deployment.yaml -n $KYMA_EXAMPLE_NS
+    ```
+
+3. Deploy the ServiceMonitor:
+
+    ```bash
+    kubectl apply -f deployment/service-monitor.yaml
+    ```
+
+### Access the exposed metrics in Prometheus
+
+1. Run the `port-forward` command on the `monitoring-prometheus` service:
+
+    ```bash
+    kubectl port-forward -n kyma-system svc/monitoring-prometheus 9090:9090
+    ```
+
+All the **sample-metrics** endpoints appear as the [`Targets`](http://localhost:9090/targets#job-sample-metrics-8081) list.
+
+2. Use either `cpu_temperature_celsius` or `hd_errors_total` in the **expression** field [here](http://localhost:9090/graph).
+3. Click the **Execute** button to check the values scrapped by Prometheus.
+
+### Cleanup
+
+Run the following commands to completely remove the example and all its resources from the cluster:
+
+1. Remove ServiceMonitor in the `kyma-system` Namespace.
+
+    ```bash
+    kubectl delete servicemonitor -l example=monitoring-custom-metrics -n kyma-system

```

Comment:
```suggestion
    kubectl delete servicemonitor -l example=monitoring-custom-metrics -n kyma-system

```
I'm not really sure, but should it also say `example=monitoring-custom-metrics-kyma1` here instead of `example=monitoring-custom-metrics`? 🤔 And the same for[ line 62](https://github.com/kyma-project/examples/pull/186/files?authenticity_token=186rP2pvPK%2B2Ucks0btvx4bY9Qn2XG5JGoGY03rCuarUdXusAGnVBD0jeNO%2FLD5A80wGPwVzmzRnifFurh146g%3D%3D&file-filters%5B%5D=.md&file-filters%5B%5D=.txt&file-filters%5B%5D=.yaml&file-filters%5B%5D=dotfile#diff-c57209b2f4b74415107e6c5be1c2d34c34858229912282a4bd7ab58efa783b72R62).

Reply:
Thank you 🙏